drltrace/CMakeLists.txt

```diff
@@ -109,3 +109,12 @@ endif ()
 install(TARGETS drltrace DESTINATION "${INSTALL_BIN}"
   PERMISSIONS ${owner_access} OWNER_EXECUTE GROUP_READ GROUP_EXECUTE
   WORLD_READ WORLD_EXECUTE)
+
+##################################################
+# drltrace test
+
+if (BUILD_TOOL_TESTS)
+  get_target_property(app_path drsyscall_app LOCATION${location_suffix})
+  get_target_property(drltrace_path drltrace LOCATION${location_suffix})
+  add_test(drltrace ${drltrace_path} -- ${app_path})
```

Comment:
All this will test is that the app didn't crash.  As a quick, initial test, sure, but we should have tests that ensure the output includes the library routines we expect drltrace to print.  Could we have comments here about adding more tests.  Also, the commit message should not say that it fixes #1960 as it is just one small exit-code test -- and thus it should also switch to the multi-commit title style (e.g., "i#1960 drltrace tests: add initial exit-code-only test").  (Remember to ensure the final squash-and-merge commit message is what we want -- editing the pull request title doesn't affect that.)